src/RpcClient.ts

```diff
@@ -102,6 +103,8 @@ class PostMessageRpcClient extends RpcClient {
         }
         await this._connect();
         window.addEventListener('message', this._receiveListener);
+        if (this._serverCloseCheckInterval !== -1) return;
```

Comment:
_Or_ you could use `null` instead of `-1`.

Reply:
I'm using -1 out of laziness because otherwise I'd need to check for `null` before calling `clearInterval` due to `null` not being an accepted input value for `clearInterval` ;)

Reply:
I think the `!` could help there: `clearInterval(this._serverCloseCheckInterval!)`.

Reply:
I'd leave it the way it currently is.
Adding the `!` would not be quite correct as `close` can be called multiple times and then `this._serverCloseCheckInterval` would actually be `null`.

Reply:
However, `null` can be passed into `clearInterval` and it will do nothing, as expected:

>(If handle does not identify an entry in the list of active timers of the WindowOrWorkerGlobalScope object on which the method was invoked, the method does nothing.)
>https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#list-of-active-timers

Reply:
Still, `!` is used to signal that a value is non-null/non-undefined.
To use it here is not nice.
I could live with `undefined` instead of `null` as the typescript type declaration accepts `number | undefined` for `clearInterval`.
But what's actually wrong with `-1`?